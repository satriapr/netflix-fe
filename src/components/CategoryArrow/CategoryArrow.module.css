.arrowPrev,
.arrowNext {
  color: white;
  font-size: 40px;
}

@media only screen and (max-width: 900px) {
  .arrowPrev,
  .arrowNext {
    font-size: 20px;
  }
}