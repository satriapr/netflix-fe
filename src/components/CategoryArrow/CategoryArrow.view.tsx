import React from 'react'

import styles from './CategoryArrow.module.css'

interface IProps {
  text: string
  className: string
}

const CategoryArrow: React.SFC<IProps> = ({ text, className }) => {
  return <div className={className}>{text}</div>
}

const ArrowLeft = CategoryArrow({ text: '<', className: styles.arrowPrev })
const ArrowRight = CategoryArrow({ text: '>', className: styles.arrowNext })

export default CategoryArrow

export { ArrowLeft, ArrowRight }
