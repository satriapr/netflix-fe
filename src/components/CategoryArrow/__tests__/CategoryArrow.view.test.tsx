import CategoryArrow from '../CategoryArrow.view'
import { testSnapshots } from '../../../utils/test.util'

describe('CategoryArrow Component', () => {
  testSnapshots(CategoryArrow, [
    {
      description: 'CategoryArrow props',
      props: {
        text: '<',
        className: 'arrow-prev'
      }
    }
  ])
})
