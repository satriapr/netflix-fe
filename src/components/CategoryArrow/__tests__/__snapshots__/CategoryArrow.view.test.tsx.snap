// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryArrow Component snapshots CategoryArrow props 1`] = `
<div
  className="arrow-prev"
>
  &lt;
</div>
`;
